### PR TITLE
Definition for Polymorphism

### PR DESCRIPTION
**Pull request by L1AMcB**
**- What I did**
Added the definition of polymorphism
**- How I did it**
created branch and used cherry pick function
**- How to verify it**
read it and let me know what you think
**- Description for the changelog**
Added definition for polymorphism

### CODE REVIEW

**Comment by L1AMcB** [comment]
I don't have how polymorphism is used within assign, just the definition and an example for now.

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
This might be what we want for Polymorphism

https://atsign.dev/docs/get-started/understand-the-platform/polymorphism/

**Comment by xavierchanth** [comment]
> This might be what we want for Polymorphism
> 
> https://atsign.dev/docs/get-started/understand-the-platform/polymorphism/

Correct, we want Atsign's definition for polymorphism for this one. You could even start with a dictionary definition and expand upon it in the context of our tech.

**Comment by JeremyTubongbanua** [comment]
Good work so far! Maybe you could relate it to namespaces because I believe namespaces and polymorphism are related.

**Review by Jesseadriano10 — CHANGES_REQUESTED** [review]
* It would be good to include the Polymorphic Data header from the article into the definition.
* Also, I would add a breadcrumb card on the related sources.

Other than that, it is looking good to me!

**Comment by xavierchanth** [comment]
Beyond @Jesseadriano10 's requested changes, LGTM

File: content/docs/reference/polymorphism.md [hunk 1]
```diff
@@ -14,4 +14,20 @@ toc: false # Change this to "true" to show the table of contents
...
+
+### Polymorphic Data
+We can even take things one step further: the [@buzz app](https://atsign.com/apps/buzz/) is a stellar example of polymorphic data on the @platform. @buzz is like your contacts application, but with a twist. Rather than manually entering details for all of your contacts, and having to deal with duplicate contact entries from trying to sync all of your contacts over the cloud, you simply connect with someone through their @sign. In @buzz, they can create and share a specific persona with you. For example, I might want to share my work phone and email with colleagues, but my personal phone and a different email with friends. I can set up separate “work” and “friends” personas with each set of information, so when I connect with someone new, all I have to do is share the correct persona with that person. Better yet, if I want to share multiple personas, I can do that too! If I move or change my email address, I can update my information in the app, and my contacts have access to the new information and don’t have to make any edits themselves.
```
Comment (xavierchanth) [inline content/docs/reference/polymorphism.md]: You can probably remove the "We can even take things one step further:"

**Review by xavierchanth — CHANGES_REQUESTED** [review]
I believe the headings used in other pages are `##`, please double check that.

**Review by xavierchanth — APPROVED** [review]
LGTM

**Review by JeremyTubongbanua — APPROVED** [review]
Good work!

**Review by Jesseadriano10 — APPROVED** [review]
Nice Job Liam!

**Pull request merged**